src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -2152,24 +3305,314 @@ namespace Google.Example.Library.V1.Samples
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+// This is a generated sample ("FlattenedStreamingBidi", "empty")
+
+// sample-metadata
+//   title: Server streaming
+//   description: Testing calling forms
+//   usage: dotnet run
+
+using CommandLine;
+
+// [START sample]
+namespace Google.Example.Library.V1.Samples
+{
+    using Google.Example.Library.V1;
+    using System;
+
+    public class StreamShelvesFlattenedStreamingBidiEmpty
+    {
+        /// <summary>
+        /// Testing calling forms
+        /// </summary>
+        public static async Task SampleStreamShelvesAsync()
+        {
+            LibraryServiceClient libraryServiceClient = LibraryServiceClient.Create();
+
+            // Initialize streaming call, retrieving the stream object
+            LibraryServiceClient.StreamShelvesStream duplexStream =
+                libraryServiceClient.StreamShelves();
+
+            // Sending requests are retrieving responses can be arbitrarily interleaved.
+            // Exact sequence will depend on client/server behavior.
+
+            // Create task to do something with responses from server
+            Task responseHandlerTask = Task.Run(async () =>
+                {
+                    IAsyncEnumerator<StreamShelvesResponse> responseStream = duplexStream.ResponseStream;
+                    while (await responseStream.MoveNext())
+                    {
+                        StreamShelvesResponse item = responseStream.Current;
+                        Console.WriteLine(item);
+                    }
+                    // The response stream has completed
+                });
+
+            // Send requests to the server
+            bool done = false;
+            while (!done)
+            {
+                // Initialize a request
+                ShelfName name = new ShelfName("[SHELF_ID]");
+                // Stream a request to the server
+                await duplexStream.WriteAsync(request);
+
+                // Set "done" to true when sending requests is complete
+                done = true;
+            }
+            // Complete writing requests to the stream
+            await duplexStream.WriteCompleteAsync();
+            // Await the response handler.
+            // This will complete once all server responses have been processed.
+            await responseHandlerTask;
+        }
+        // [END sample]
+        public static void Main(string[] args)
+        {
+            SampleStreamShelves();
```

Comment:
Need to call `SampleStreamShelvesAsync`.

Reply:
Done.